### ISSUE TITLE
[demo] Add alt-text to code and examples

### ISSUE THREAD

**Issue by stp-ip** [okkur/syna#400]
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently we don't always provide an alternative `text` in our examples.

**What you expected to happen**:
For accessibility and missing images a `text` on icons and images would be helpful.

**Comment by mpourismaiel** [comment]
How would text on icons work?
